setup.cfg

```diff
@@ -2,9 +2,13 @@
 name = meson
 version = attr: mesonbuild.coredata.version
 description = A high performance build system
+long_description = file: README.md
+long_description_content_type = text/markdown
```

Comment:
This is not mentioned in the PR description or the commit message. Also, we have an existing long_description, and with your PR we have two of them and setuptools emits a hard error, so this could never build.

Please remove the long_description change.